### DISCUSSION TITLE
Operators selection for both teams

### DISCUSSION THREAD

**Discussion by Sleepy-Git-User** [redraskal/r6-dissect#24] [Q&A] [answered]
In the example json it appears only 1 teams operators are listed. Is this  updated or can you only get one teams operators ? 

Edit: It appears only Defenders are listed currently, with attackers not appearing at all even as players occasionally

**Comment by redraskal** [comment]
It's issue #10. This still needs to be reverse engineered

**Comment by redraskal** [comment, marked as answer]
This is part of the latest release
